views/pre-install.php

```diff
@@ -66,7 +66,22 @@
         Your server is missing the required <code>json</code> extension.
         <span>
             Please contact your webhost technical support and ask them to make sure
-            you are on a build of PHP containing the <cod>json</cod> extension.
+            you are on a build of PHP containing the <code>json</code> extension.
+        </span>
+    </li>
+    <?php } ?>
+
+    <?php if ($mysqlCompatible) { ?>
+    <li class="good">
+        Your server has proper MySQL permissions.
+    </li>
+    <?php } else { ?>
+    <li class="bad">
+        Your MySQL permissions are incorrect.
+        <span>
+            Please contact your webhost technical support and ask them to increase
+            your database permissions to include <code>DROP</code> and <code>ALTER</code>
+            privileges.
```

Comment:
can you add some helpful info here exposing the  _db user_ and _db name_? like...

```HTML
... to increase the permissions of database user <code>jim_jam</code> for
the database <code>lol_wp1</code> to include ...
```

using the WordPress constants